### PR TITLE
306 replace guava mediatype

### PR DESCRIPTION
**Pull request by gnl42**
Replace guavas MediaType

two down, one to go

### CODE REVIEW

**Review by ruspl-afed — APPROVED** [review]
LGTM

File: org.eclipse.mylyn-target/org.eclipse.mylyn.target.target
Comment (ruspl-afed) [inline org.eclipse.mylyn-target/org.eclipse.mylyn.target.target:1]: It would be great to reduce the diff for this file

Comment (gnl42) [reply org.eclipse.mylyn-target/org.eclipse.mylyn.target.target:1]: How so? It's just one line added

Comment (ruspl-afed) [reply org.eclipse.mylyn-target/org.eclipse.mylyn.target.target:1]: I think target editor did it

**Comment by gnl42** [comment]
@ruspl-afed Any chance you have 'ignore white space' disabled?
image

**Comment by ruspl-afed** [comment]
> @ruspl-afed Any chance you have 'ignore white space' disabled? 

Yes, this is how I was able to look at the distilled change.

**Comment by merks** [comment]
FYI, the changes here introduced a new dependency on org.apache.tika.core which needs to be in the update site:

https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/518

**Comment by ruspl-afed** [comment]
> FYI, the changes here introduced a new dependency on org.apache.tika.core which needs to be in the update site:
> 
> #518

Right, I forgot that we need to add every introduced 3rd party bundle to category.xml

**Comment by gnl42** [comment]
Go for 2.x

**Comment by ruspl-afed** [comment]
Go for 2.x

**Comment by gnl42** [comment]
@merks Could you remind me how I can get apache commons-text added to Orbit?

**Comment by merks** [comment]
Open an issue here and ask nicely 😇 

https://github.com/eclipse-orbit/orbit-simrel/issues

**Comment by merks** [comment]
Of course it pays off to see what's already there:

image

The tabular summary is quite helpful:

https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/table.html

**Comment by gnl42** [comment]
Assuming one even remembers that there is a milestone :)

The summary is exactly what I was about to ask for

Thanks

**Pull request merged**